WooCommerce/Classes/ViewRelated/Dashboard/Stats v4/StoreStatsAndTopPerformersViewController.swift

```diff
@@ -411,6 +416,36 @@ private extension StoreStatsAndTopPerformersViewController {
         }
     }
 
+    @MainActor
+    func configureCustomRangeTab() async {
+        guard featureFlagService.isFeatureFlagEnabled(.customRangeInMyStoreAnalytics) else {
+            return
+        }
+
+        guard let customRange = await loadCustomRangeTab() else {
+            return
+        }
+
+        createCustomRangeTab(range: customRange)
+    }
+
+    @MainActor
+    func loadCustomRangeTab() async -> StatsTimeRangeV4? {
```

Comment:
nit: Consider renaming this into `loadTimeRangeForCustomRangeTab`. IMO, this helps to clarify that we are not loading the actual tab in this method.